spec-main/api-remote-spec.ts

```diff
@@ -222,6 +223,74 @@ ifdescribe(features.isRemoteModuleEnabled())('remote module', () => {
     });
   });
 
+  describe('nativeImage serialization', () => {
+    const w = makeWindow();
+    const remotely = makeRemotely(w);
+
+    it('can serialize an empty nativeImage', async () => {
+      const getEmptyImage = (img: nativeImage) => {
+        return new Promise((resolve, reject) => {
+          if (img.isEmpty()) {
+            resolve('empty image');
+          } else {
+            reject(new Error('incorrectly serialized image'));
+          }
+        });
```

Comment:
It seems weird that this is a promise because it doesn't do any asynchronous things—can it be refactored to not use a promise? e.g.
```suggestion
        return img.isEmpty()
```